rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
```

Comment:
- [ ] Kindly ensure that rental is adding itself to ``` @person and @book ``` to achieve the belongs-to association. e.g
`book.rentals << self`